tutorials/https-load-balancing-nginx/index.md

```diff
@@ -352,7 +352,8 @@ balancer:
 
         server {
           listen 80;
-          return 301 https://$host$request_uri;
+           if ($scheme = "http") {
```

Comment:
unmatched opening bracket

Reply:
here we have the closing bracket. reference - [NGINX](https://www.nginx.com/resources/wiki/start/topics/examples/SSL-Offloader/#redirector-server)
here is the [screenshot](https://i.ibb.co/4jjcL9s/Screenshot-2021-07-05-at-10-31-15-PM.png)

Reply:
In your modification, the `server` block's opening bracket is now left unmatched.

Reply:
My bad, I missed the closing brackets for the ``server`` block, now fixed.

Reply:
Thanks for the fix.